docker/venice-client/fetch.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+router=$1
+storeName=$2
+key=$3
+java -jar /opt/venice/bin/venice-thin-client-all.jar $storeName $key $router false ""  2>/dev/null
```

Comment:
Is this `fetch.sh` going to replace the query store script we have? If so, might worth to update the readme for the quickstart guide? (For some reasons, I just cannot find the file anymore

Reply:
Yes, and yes that readme will update once we refresh images on dockerhub